packages/devtools_app/lib/src/primitives/feature_flags.dart

```diff
@@ -2,15 +2,27 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+/// If true, features under construction will be enabled.
+///
+/// By default, the constant is false.
+/// To enable it, pass the compilation flag
+/// `--dart-define=enable_experiments=true`.
+///
+/// To enable the flag in debug configuration of VSCode, add value:
+///   "args": [
+///     "--dart-define=enable_experiments=true"
+///   ]
+const bool _kEnableExperiments = bool.fromEnvironment('enable_experiments');
```

Comment:
would this enable an end user to hack these experiments on? end user being someone running a flutter app and debugging with devtools

Reply:
It is compile time flag. If end user compiles devtools, then yes, otherwise - no.

Do we want them to hack?

Reply:
No we don't. Would there ever be an occasion where we don't want all the experiments enabled at once?

Reply:
Or we can give user build that was compiled with this flag.

Reply:
> No we don't. Would there ever be an occasion where we don't want all the experiments enabled at once?

I do not think it can happen. If it does, we will adjust the code.